testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

```diff
@@ -325,7 +325,7 @@ Feature: PXE boot a Retail terminal
     When I follow "pxeboot" terminal
     And I follow "Details" in the content area
     And I follow "Connection" in the content area
-    Then I should see "proxy" short hostname
+    Then I should see "proxy.example.org" short hostname
```

Comment:
shouldn't you remove " short hostname" ? Look at Vanda's PR

Reply:
Correct. My brain missed that. Thank you and fixed.